src/components/MultiLink/Legend.vue

```diff
@@ -33,6 +33,14 @@ export default Vue.extend({
     return {
       svgHeight: 50,
       yAxisPadding: 25, // Gives enough width for hundreds on the y axis
+      stickyBarHeight: 100,
+      stickyBarHorizSpacing: 60,
+      stickyBarWidth: 30,
+      stickyColorMapSquareSize: 15,
+      stickyPadding: 15,
+      stickyPlusBackgroundSize: 30,
+      stickyRowHeight: 50,
+      stickyVarNameIndent: 50,
```

Comment:
Might it be easier to collect these into a single object named `sticky`?

Reply:
See 3789364